netlogo-gui/resources/system/behaviorspace.dtd

```diff
@@ -11,6 +11,7 @@ section of a NetLogo model file.
 <!ATTLIST experiment
   name CDATA ""
   repetitions CDATA "1"
+  sequentialRunOrder CDATA #IMPLIED
```

Comment:
Sorry I didn't catch this before. Is it possible to define this as `(true|false) "true"` (the same as `runMetricsEveryStep` below)?